Sources/MapboxNavigation/InstructionLabel.swift

```diff
@@ -46,7 +47,7 @@ open class InstructionLabel: StylableLabel, InstructionPresenterDataSource {
                                              dataSource: self,
                                              traitCollection: customTraitCollection ?? traitCollection,
                                              downloadCompletion: update)
-        
+        styleID = presenter.spriteRepository.styleID
```

Comment:
Because developers may use their own style and it has the unpredictable Id name. The styleType is also related with different platform, such as CarPlay or mobile. 

The solution should based on using other highlighted color for border and text in exit views and generic route shields, instead of the `Style` .